chain-extensions/pallet-assets/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "pallet-chain-extension-assets"
+version = "0.1.0"
+authors = ["Stake Technologies <devops@stake.co.jp>"]
```

Comment:
Aren't we? :slightly_smiling_face:

Reply:
Not sure why, but I read this as 'Parity' before :see_no_evil: 

Sorry for the confusion and incorrect comment, you were correct in the first place.